### DISCUSSION TITLE
Merge with bookmarks adds .pdf by default

### DISCUSSION THREAD

**Discussion by laenzlinger** [pdfcpu/pdfcpu#848] [General]
I was wondering if it would make sense to remove .pdf from the bookmarks. For me it would be a good default, because when opening a PDF, its makes sense that the contents are also PDF files.

WDYT

**Comment by laenzlinger** [comment]
It would not be difficult to implement, I could create a PR, but how would one influence it?

There is the `CreateBookmarks` flag in the Configuration.

This flag could be extended to support 3 values:

- None
- Filename
- Filename.ext

**Comment by laenzlinger** [comment]
In the meantime I am trying to implement a workaround by export - modify - import the bookmarks

**Comment by hhrutter** [comment]
Are you referring to the bookmark/outline titles created for each merge input file?

**Reply by laenzlinger** [reply]
yes, exactly, the bookmarks created during the merge process

**Comment by hhrutter** [comment]
closed due to inactivity.
